ot3_state_manager/Makefile

```diff
@@ -0,0 +1,39 @@
+python := poetry run python
+pytest := poetry run py.test
+SHX := npx shx
```

Comment:
definitely do not use npx here, we can use normal shell

Reply:
2b32951e169f0fc8559a78f4f1188103dfad43f2